.goreleaser.yml

```diff
@@ -1,76 +1,95 @@
 # GoReleaser config
 
+---
 before:
   hooks:
     - go mod tidy
 
 builds:
-- id: pop_darwin
-  binary: soda
-  main: soda/main.go
-  goos:
-    - darwin
-  goarch:
-    - amd64
-  env:
-    - CGO_ENABLED=1
-    - CC=o64-clang
-    - CXX=o64-clang++
-  flags:
-    - -tags
-    - sqlite
+  - id: pop_darwin
+    binary: soda
+    main: soda/main.go
+    goos:
+      - darwin
+    goarch:
+      - amd64
+    ldflags:
+      - -s
+    env:
+      - CGO_ENABLED=1
+      - CC=/osxcross/target/bin/o64-clang
+      - CXX=/osxcross/target/bin/o64-clang++
+    flags:
+      - -tags
+      - sqlite
 
-- id: pop_linux
-  binary: soda
-  main: soda/main.go
-  env:
-    - CGO_ENABLED=1
-  flags:
-    - -tags
-    - sqlite
-  goos:
-    - linux
-  goarch:
-    - amd64
-    - 386
+  - id: pop_darwin_arm64
+    binary: soda
+    main: soda/main.go
+    goos:
+      - darwin
+    goarch:
+      - arm64
+    ldflags:
+      - -s
+    env:
+      - CGO_ENABLED=1
+      - CC=/osxcross/target/bin/oa64-clang
+      - CXX=/osxcross/target/bin/oa64-clang++
+    flags:
+      - -tags
+      - sqlite
 
-- id: pop_windows_i686
-  binary: soda
-  main: soda/main.go
-  ldflags:
-    - "-extldflags '-static'"
-  env:
-    - CGO_ENABLED=1
-    - CC=i686-w64-mingw32-gcc
-    - CXX=i686-w64-mingw32-g++
-  flags:
-    - -tags
-    - sqlite
-  goos:
-    - windows
-  goarch:
-    - 386
+  - id: pop_linux
+    binary: soda
+    main: soda/main.go
+    env:
+      - CGO_ENABLED=0
```

Comment:
I changed `CGO_ENABLED=0` on the Linux builds and it built fine in https://github.com/jfreeland/pop/runs/6069504394?check_suite_focus=true#step:5:95.  But maybe that is not the correct step here.